internal/data/organizations.go

```diff
@@ -203,6 +214,17 @@ func (om *OrganizationModel) Update(ctx context.Context, ou *OrganizationUpdate)
 		}
 	}
 
+	if ou.PrivacyPolicyLink != nil {
+		if *ou.PrivacyPolicyLink != "" {
+			link, _ := url.ParseRequestURI(*ou.PrivacyPolicyLink)
+			fields = append(fields, "privacy_policy_link = ?")
+			args = append(args, link.String())
+		} else {
+			// When empty value is passed by parameter we set the DEFAULT value for the column.
+			fields = append(fields, "privacy_policy_link = DEFAULT")
```

Comment:
Adding `= DEFAULT` makes sense for `OTPMessageTemplate` or `SMSRegistrationMessageTemplate`, because those columns do have a valid default. 
```
sms_registration_message_template VARCHAR(255) NOT NULL DEFAULT 'You have a payment waiting for you from the {{.OrganizationName}}. Click {{.RegistrationLink}} to register.',
```

In the case of of `privacy_policy_link ` it seems like your default is null so this is not needed.